src/main/java/pro/beam/api/resource/chat/events/data/IncomingMessageData.java

```diff
@@ -17,6 +17,7 @@
     @SerializedName("user_id") public int userId;
     @SerializedName("user_roles") public List<BeamUser.Role> userRoles;
     public MessageComponent message;
+    public String target;
```

Comment:
The target is actually somewhat extraneous data (and it is redundant). It might be trimmed out in the future; it probably should not be exposed.